### PR TITLE
PR: Fix incorrect custom shortcut displayed in menus for Run Cell command

### LINKED ISSUE #8076: Incorrect custom shortcut displayed in menus for Run Cell command
## Problem Description

I've defined a custom shortcut, Ctrl-Shift-Return, for "Run Cell" (along with the other run commands). While other shortcuts display fine, "Run Cell" displays the shortcut as ``Ctrl-Return`` in the Run menu instead, which I believe is (perhaps not coincidentally) the original shortcut. However, the correct shortcut does work just fine.

This is the case in both Spyder 3 and Spyder 4. Ctrl-Shift-Return is the correct shortcut that I set, and indeed works, but it displays as Ctrl-Return... There should be enough space to do so, considering it adds only one extra small character over Alt-Shift-Return that is displayed, and there's a ton of extra horizontal space in the menu. Do you want me to run some more tests on this?

## Versions

* Spyder version: 3.3.1 and ``master``
* Python version: 3.6.6
* Qt version: 5.9.6
* PyQt version: 5.9.2
* Operating System name/version: Windows 8.1 x64

### PR DESCRIPTION
**Pull request by steff456**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [ ] Added unit test(s) covering the changes (if testable)

* [x] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

Verify the displayed shortcut in menus for `run cell` and `run cell and advance` commands. 

image

### Issue(s) Resolved

Fixes #8076

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: steff456

### CODE REVIEW

File: spyder/config/main.py [hunk 1]
```diff
@@ -427,11 +421,11 @@
               'editor/zoom reset': "Ctrl+0",
               'editor/close file 1': "Ctrl+W",
               'editor/close file 2': "Ctrl+F4",
-              'editor/run cell': RUN_CELL_SHORTCUT,
-              'editor/run cell and advance': RUN_CELL_AND_ADVANCE_SHORTCUT,
+              'editor/run cell': 'Meta+Return' if MAC else 'Ctrl+Return',
```
Comment (goanpeca) [inline spyder/config/main.py]:
```suggestion
              'editor/run cell': CTRL + '+Return',
```

**Comment by steff456** [comment]
I'm currently checking the apply function in preferences. With the actual changes the shortcut changes after restrarting spyder but not live.

**Comment by ccordoba12** [comment]
@steff456, please merge with master to fix the current merge conflict and revert the `logger.debug` lines you added.

I think what you've done here is fine. Let's leave for the future to see how we could update live the shortcuts present in menus.

**Comment by steff456** [comment]
I'm creating a new PR because some commits were deleted during merge.

**Pull request closed without merging**